src/components/Card.tsx

```diff
@@ -88,13 +91,24 @@ const cardContainer = (
       'py-5': paddingType === 'normal',
       'py-8': typeof paddingType === 'undefined',
     }),
-    width(
-      thin ? 'sm:!w-thin-card' : 'sm:w-card',
-      thin ? 'tiny:w-thin-mobile' : 'w-mobile-card',
-      { 'w-32': thin }
-    ),
+
+    bigCard
```

Comment:
We need to simplify this logic, I think there should be a way

Reply:
I think I've simplified it a bit, wdyt @T-Damer ?